package.json

```diff
@@ -18,17 +18,19 @@
     "react-dom": "^0.14.3"
   },
   "devDependencies": {
+    "babelify": "^7.3.0",
+    "browserify": "^14.4.0",
     "gulp": "^3.9.1",
-    "gulp-concat": "^2.6.0",
-    "gulp-concat-css": "^2.2.0",
-    "gulp-es6-transpiler": "^1.0.1",
-    "gulp-jsx": "^2.0.1",
-    "gulp-minify": "0.0.5",
+    "gulp-connect": "^2.3.1",
+    "gulp-livereload": "^3.8.1",
     "gulp-minify-css": "^1.2.2",
     "gulp-rename": "^1.2.2",
     "gulp-sass": "^2.1.0",
-    "gulp-typescript": "^3.1.3",
+    "gulp-sourcemaps": "^2.6.0",
+    "gulp-uglify": "^3.0.0",
     "gulp-watch": "^4.3.11",
-    "typescript": "^2.0.10"
+    "react": "^0.14.9",
```

Comment:
`react` should be a dependency, not a devDependency. Moreover, you already have `react` in dependendenices list.